run.sh

```diff
@@ -0,0 +1,52 @@
+
+#!/bin/sh
+#
+# Run Ylitse app on a device.
+
+#set the API_URL variable
+if [[ ${YLITSE_API_URL} && ${YLITSE_LOGIN_URL} && ${YLITSE_TERMS_URL} && ${YLITSE_USERGUIDE_URL} && ${YLITSE_FEEDBACK_URL} ]]; then
+    API_URL=${YLITSE_API_URL}
+    LOGIN_URL=${YLITSE_LOGIN_URL}
+    TERMS_URL=${YLITSE_TERMS_URL}
+    FEEDBACK_URL=${YLITSE_FEEDBACK_URL}
+    USERGUIDE_URL=${YLITSE_USERGUIDE_URL}
+else
+    API_URL="localhost"
+    LOGIN_URL="localhost"
+    TERMS_URL="localhost"
+    FEEDBACK_URL="localhost"
+    USERGUIDE_URL="localhost"
+fi
+
+set -eu
+
+print_help() {
+    echo "Usage: ./run {android|ios}"
+}
+
+FILE=config.json
+if test -f "$FILE"; then
+    echo "$FILE exists"
+else
+    JSON_STRING='{"baseUrl":"'"$API_URL"'", "loginUrl":"'"$LOGIN_URL"'", "feedBackUrl":"'"$FEEDBACK_URL"'", "termsUrl":"'"$TERMS_URL"'", "userGuideUrl":"'"$USERGUIDE_URL"'"}'
+    echo $JSON_STRING > $FILE
+    echo "Created new $FILE with api_url=$API_URL"
+fi
+
+if [ $# -ne 1 ]; then
+    print_help
+fi
+
+if [ $# -ge 1 ] && [ -n "$1" ]; then
+    case "$1" in
+        ios)
+            npx react-native run-ios
+            ;;
+        android)
+            npx react-native run-android
+            ;;
+        *)
+            print_help
+            ;;
+    esac
+fi
```

Comment:
Should this actually just configure the build and not at all run the app.

I am thinking a situation where one would like to build an ios or android packake. They would first need to run this script and then build it.

I feel that it could make more sense to have a config script that just sets the URLs.

Ofc. build scripts can be made separately but they both have to config the build

Reply:
OR should we have separate config run and build scripts. Could be a case for a new PR? : )